sample-queries.csv

```diff
@@ -22,7 +22,7 @@ Users,POST,create user,/v1.0/users,,https://developer.microsoft.com/en-us/graph/
         ""mailNickname"": ""MelissaD"",
         ""passwordPolicies"": ""DisablePasswordExpiration"",
         ""passwordProfile"": {
-            ""password"": ""Test1234"",
+            ""password"": ""<Enter Password>"",
```

Comment:
Oh darn, a password is required for a new user. The problem with is that it is still a default password. I think we will need to generate a unique token for each user so that the password is not predictable.

https://github.com/microsoftgraph/microsoft-graph-explorer/blob/dev/src/app/tokens.ts